ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Image from '#app/components/Image';
+import { createSrcsets } from '#app/lib/utilities/srcSet';
+import getOriginCode from '#app/lib/utilities/imageSrcHelpers/originCode';
+import getLocator from '#app/lib/utilities/imageSrcHelpers/locator';
+import styles from './styles';
+
+const MaskedImage = ({
+  imageUrl,
+  imageUrlTemplate,
+  imageWidth,
+}: {
+  imageUrl: string;
+  imageUrlTemplate: string;
+  imageWidth: number;
+}) => {
+  const { dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  const url = imageUrlTemplate.split('{width}')[1];
+
+  const originCode = getOriginCode(url);
+  const locator = getLocator(url);
+
+  const { primarySrcset, primaryMimeType, fallbackSrcset, fallbackMimeType } =
+    createSrcsets({
+      originCode,
+      locator,
+      originalImageWidth: imageWidth,
+    });
+
+  return (
+    <div css={isRtl ? styles.maskedImageRtl : styles.maskedImageLtr}>
+      <Image
+        alt=""
+        src={imageUrl}
+        aspectRatio={[16, 9]}
+        srcSet={primarySrcset || undefined}
+        fallbackSrcSet={fallbackSrcset || undefined}
+        mediaType={primaryMimeType || undefined}
+        fallbackMediaType={fallbackMimeType || undefined}
+        sizes="(min-width: 1008px) 660px, 100vw"
```

Comment:
I'm not sure this is best approach since it's hardcoded. 

As I understand it, this means for any screen sizes smaller than 1008px I would expect it to render at the srcSet closest to the viewport width. And then anything 1008px and above it would render at the srcSet closed to 660px..